\documentclass[thesis.tex]{subfiles}

\title{Estimating duration in the presence of misclassification}
\author{Joshua Blake}
\date{\today}

\begin{document}

\ifSubfilesClassLoaded{
  \setcounter{chapter}{1}
}

\chapter{SARS-CoV-2: biology and data} \label{biology-data}

\todo[inline]{I should maybe swap 2.1 and 2.2 order. I like the current order because natural history feels more basic and introductory, but it does reference a bunch of stuff about PCR testing so it might be easier to follow the other way round.}

\todo[inline]{Rewrite chapter 2 intro after writing thesis intro because it will be clearer how to divide content.}

SARS-CoV-2 is a beta coronavirus that emerged in late 2019.
It is closely related to previous coronaviruses that have caused outbreaks in humans, SARS-CoV and MERS-CoV.
\todo{Cite and check how SARS-CoV-2 is related to SARS-CoV and MERS-CoV, and other human coronaviruses.}

Coronaviruses are RNA (Ribonucleic acid) viruses.
RNA viruses use RNA, rather than DNA, as their genetic material.
\todo{why is being a RNA virus important?}

\section{Natural history of the disease} \label{biology-data:sec:natural-history}

\begin{figure}
  \makebox[\textwidth][c]{
  \begin{tikzpicture}[
      node distance = 4cm,
      on grid,
      auto,
      ->,>=stealth',
      every state/.style={draw,rectangle,align=center},
      ]

      \node[state] (pre) {PCR negative \\ Not infectious};
      \node[state, right of=pre] (pos) {PCR positive \\ Not infectious};
      \node[state, right of=pos] (infectious) {PCR positive \\ Infectious};
      \node[state, right of=infectious] (not-infectious) {PCR positive \\ Not infectious};
      \node[state, right of=not-infectious] (neg) {PCR negative \\ Not infectious};

      \node[draw=none, below=2cm of pre] (infected) {} ;
      \node[draw=none, below=2cm of neg] (recovered) {} ;

      \node[above=2cm of pre] (titleA) {(A)};
      \node[below=1cm of infected] (titleB) {(B)};

      % \draw (0,-2) -- node {Infected} (pre);
      % \draw (not-infectious) -- (0,-2);

      \path (infected) edge node {Infected} (pre)
          (pre) edge (pos)
            (pos) edge (infectious)
            (infectious) edge (not-infectious)
            (not-infectious) edge (neg)
            (neg) edge (recovered);
    
    % Detectable
    \node[below = 1cm of infectious] (detect-label) {Detectable};
    \node[draw, thick, inner sep=0.3cm, fit=(pos) (infectious) (not-infectious) (detect-label), fill=blue!10,opacity=0.2] (detect-box) {};
  \end{tikzpicture}
  }
  \centering\includegraphics[width=\textwidth]{biology-data/natural-history}
  \caption[Natural history of SARS-CoV-2.]{%
    Natural history of SARS-CoV-2.
    (A) A simplified state-based model, which I generally use in this thesis.
    Shading represents states where the individual is detectable by a PCR test.
    (B) A full model of the natural history, including continuous processes (Reproduced from \textcite{puhachSARSCoV2} with permission; durations are based on averages, there is a large amount of individual variation).
    PCR positive refers to the main diagnostic test used in this thesis, and detect RNA genomes (see \cref{biology-data:sec:PCR} for more details).
    Ag-RDT (antigen rapid diagnostic test) tests are lateral flow tests, which detect infectious virus rather than RNA, and are not used in this thesis.
  }
  \label{biology-data:fig:natural-history}
\end{figure}

An individual infected with SARS-CoV-2 goes through several stages (see \cref{biology-data:fig:natural-history}).
\todo{Find citations for disease stages.}
First, they are exposed to the virus and become infected.
At this initial stage they are not infectious (cannot spread the disease to others), do not have symptoms, and will not test positive on a PCR test.
After a few days, they will start testing positive on a PCR test; this is normally prior to being infectious or having symptoms.
Next, the individual becomes infectious; normally, this occurs prior to symptom onset for those who experience symptoms.
Recovery from symptoms occurs after a few days, around the time the individual is no longer infectious.
Finally, the individual will test negative on a PCR test.

\todo[inline]{Maybe move the following paragraph because it is referencing forward a bunch of concepts not yet introduced properly, or if sections 2.1 and 2.2 are swapped it starts making sense.}
I will refer to individuals between the time they start testing positive on a PCR test and the time they stop testing positive on a PCR test as \emph{detectable}.
It is not possible to directly observe whether an individual is detectable or not due to noise in the PCR test (see \cref{biology-data:sec:PCR}).
However, viewing the number of RNA genome copies in an individual as a latent process this can then be defined as the time when the number of RNA genome copies is above the limit of detection of the PCR test.

The time for each of these stages depends on various features of host and virus characteristics.
In this thesis, I focus on pre-Alpha variants of SARS-CoV-2 in unvaccinated individuals.
The time from infection to infectiousness is known as the \emph{latent period} and is typically around\todo{cite latent period}.
The time from infection to experiencing symptoms is known as the \emph{incubation period} and is typically around\todo{cite incubation period}.
The latent period is undefined for asymptomatic individuals, around\todo{cite proportion asymptomatic} of those infected.

The above description gives three possible ways to define an individual who is infected with SARS-CoV-2: based on their PCR test result, whether they are infectious, or whether they are symptomatic.
In this thesis, I adopt a definition of infection based on PCR test results.
This is because it coincides with the data collected on the studies I use (see \cref{biology-data:sec:studies}).

A host's \emph{viral load}, the quantity of viral particles in a host, is the underlying biological process determining many of these changes~\autocite{puhachSARSCoV2}.
Any host's viral load is determined by the interaction of many biological processes, most prominently the virus's replication increasing the viral load and the host's immune response reducing the viral load~\autocite{keVivo}.
Viral replication requires the virus to infect a host cell and then use the host cell's machinery to produce more virus; as more host cells are infected this process slows because there are fewer remaining host cells, known as \emph{target cells}, to infect~\autocite{keVivo}.
The host's immune system can slow this process by various mechanisms; the most relevant for SARS-CoV-2 is by making some target cells less susceptible to infection.
A host's immune response is triggered by the viral load, and becomes stronger as the viral load increases.
A high viral load is associated with being more infectious, more likely to test positive on a PCR test, and the start of symptoms~\autocite{puhachSARSCoV2,keVivo}.


% Possible summaries
% \begin{enumerate}
%   \item \url{https://www.nature.com/articles/s41579-022-00822-w/figures/2}
%   \item \url{https://www.nature.com/articles/s41576-021-00360-w/figures/1}
%   \item \url{https://academic.oup.com/view-large/figure/305617142/ciaa1442_fig2.jpg}
% \end{enumerate}

\section{PCR testing} \label{biology-data:sec:PCR}

\todo[inline]{Consider finding or making a figure to explain PCR testing.}

PCR (Polymerase Chain Reaction) is a procedure that produces millions to billions of copies of a specific segment of DNA (Deoxyribonucleic acid) from a single copy~\autocite{smithPCR,garibyanPCR}.
This is known as \emph{amplifying} the DNA.
PCR has many uses within the biological sciences; Mark R.\ Hughes, deputy director of the Human Genome Project, has claimed it is the most important scientific technology of the last hundred years~\autocite{powledgePCR}.
It is central to the work in this thesis because it can detect the presence of SARS-CoV-2 in a sample taken from a human, most commonly a nose and/or throat swab.

SARS-CoV-2 is an RNA virus, but PCR amplifies DNA.
Therefore, before the PCR process can be used to detect SARS-CoV-2, the RNA must be converted to DNA.
This process is known as \emph{reverse transcription}.
In reverse transcription, a naturally-occurring enzyme known as reverse transcriptase generates cDNA (complementary DNA) from the RNA in the sample~\autocites{valasekPower}.
Any sequence of cDNA has a one-to-one correspondence with a sequence of RNA.
The cDNA can then be used in the standard PCR process.

PCR consists of a series of cycles, each cycle containing a three-step process~\autocite{powledgePCR,garibyanPCR}.
The first step of the PCR cycle is \emph{denaturation}.
In denaturation, the DNA is heated to separate the two strands of DNA.
The second step is \emph{annealing}.
In annealing, the temperature is lowered to allow primers to bind to the DNA.
Primers are short sequences of DNA that are complementary to the DNA sequence of interest.
By choosing the primers, the PCR process targets a portion of DNA starting and ending with a specified sequence.
The final step is \emph{extension}.
In extension, the temperature is raised to allow the enzyme DNA polymerase to extend the primers and hence replicate the DNA sequence of interest.
In the course of such a cycle, the amount of DNA doubles.
Therefore, the number of DNA copies increases geometrically with the number of cycles completed.

Quantitative real-time PCR (or RT-PCR) gives a measurement of the amount of the target DNA sequence present in the original sample~\autocite{yangPCRdiagnostics}.
After each PCR cycle, the amount of target DNA is measured and compared to a pre-derived standard.
Once the measured amount of DNA exceeds the standard, the target DNA is said to be detected.
The number of cycles required to reach detection is known as the \emph{cycle threshold} (Ct) value.
Quantitative real-time PCR have a number of other practical advantages over previous PCR techniques (\eg speed); see \textcite{yangPCRdiagnostics,valasekPower} for more details.
All RT-PCR tests I consider in this thesis are quantitative real-time RT-PCR tests, and produce Ct values.

The Ct value decreases as the amount of DNA present in the original sample increases.
If there is more RNA present, more DNA is created, and hence fewer cycles are required to reach detection.
The logarithm is because the process is geometric, and hence the number of DNA copies increases linearly on the log-scale each cycle.
Therefore, the Ct value is proportional to the negative log of the amount of DNA present in the original sample.
The amount of cDNA produced by reverse transcription is proportional to the amount of RNA present in the original sample.
Therefore, when testing for SARS-CoV-2, the Ct value is proportional to the negative log of the amount of SARS-CoV-2 RNA present in the original sample.

Like any diagnostic test, PCR tests are not perfect.
False positives and false negatives can occur.
The \emph{specificity} of a test is the probability of a negative test result given that the individual is not infected.
The \emph{sensitivity} of a test is the probability of a positive test result given that the individual is infected.

False positives are very rare for PCR tests.
In summer 2020, the CIS performed X PCR tests and found only Y positives, lower-bounding the test specificity at.
\todo{Insert actual numbers of CIS bounding specificity.}

False negatives are more common.
Early estimates of the sensitivity of PCR tests were around X\%; I produce estimates in \cref{E-ATACCC}.
False negatives could occur for a variety of reasons, most commonly that an individual swabs themselves poorly and hence the concentration of virus on the swab is too low to be detected by the PCR process.
Plausibly, a variety of other reasons could also lead to a false negative such as logistical issues or mislabelling of samples.
\todo{Cite reasons false negative results occur.}

The minimum amount of virus required to be detected by a PCR test is known as the \emph{limit of detection}.
\todo{Cite limit of detection and explain a bit more about it.}

Viral loads are infrequently measured exactly.
The most common form of data obtained is cycle threshold (Ct) values from real-time quantitative  reverse-transcriptase polymerase chain reaction (PCR) tests.
The Ct value is proportional to the negative log of the viral load which is convenient when modelling log viral load as a piecewise linear function, since Ct values themselves are piecewise linear under this assumption.
When a host's viral load is too low (below the limit of detection of the PCR test), no Ct value is available and hence viral load can be considered censored at the limit of detection.

Ct values cannot be transformed into viral loads (expressed in units such as the number of viral RNA copies per swab) without additional information~\autocite{dahdouhCt,puhachSARSCoV2}.
Ct values are test-specific, and can even vary between different batches of the same test.
Testing a series of samples with known viral loads allows the relationship between Ct and viral load to be estimated for a specific setup.

RT-PCR enables the detection of SARS-CoV-2 viral RNA in a sample.
However, this does not necessarily mean that the virus is viable (\ie able to reproduce)~\autocite{puhachSARSCoV2}.
High viral loads (low Ct values) are more likely to be viable, but more sophisticated tests are required to determine viability.
The gold standard procedure is attempting to grow and recover viral material from a sample, a process known as \emph{cell culturing}~\autocite{singanayagamDuration,puhachSARSCoV2,hakkiOnset}.


\section{Studies used in this thesis} \label{biology-data:sec:studies}

In this thesis, I make use of data from two studies which recruited participants from the general population in the UK.
Unlike many studies of SARS-CoV-2, these studies are not focused on specific groups such as healthcare workers or hospital patients.
Therefore, the results from these studies are more generalisable to the entire UK population.
The first study, ATACCC (Assessment of Transmission and Contagiousness of COVID-19 in Contacts) performed daily testing of a small number of individuals with limited follow-up.
The density of this sampling allows for reliable estimates of the viral load dynamics and other quantities of interest, most importantly the duration of detectability, for but the results are limited because the small sample size and short follow-up means that the tails of these distributions cannot be estimated.
The second study, the CIS (Coronavirus (COVID-19) Infection Survey), performed weekly or monthly testing of a large number of individuals with long follow-up.
The large sample size and long follow-up allows for reliable estimates of the tails of the distributions of the quantities of interest, but the results are limited because the sparse sampling means that the viral load dynamics cannot be estimated.
In particular, the sparse sampling means that the vast majority of detected infections had only one or two positive tests, and hence the viral load trajectory of these infections cannot be estimated.

\subsection{Assessment of Transmission and Contagiousness of COVID-19 in Contacts}

The ATACCC (Assessment of Transmission and Contagiousness of COVID-19 in Contacts) study was a longitudinal study of contacts reported to the NHS Test and Trace contact tracing service~\autocite{hakkiOnset}.
Contacts were invited to participate if they were 5 years or older and  could be reached within 5 days of the index case's symptom onset date.
ATACCC was formed of two waves: ATACCC1 enrolled 393 contacts between 13th Sep 2020 and 31st Mar 2021 (when pre-Alpha and Alpha variants were circulating) and ATACCC2 enrolled 345 contacts between 24th May 2021 and 228th Oct 2021 (when the Delta variant was circulating).
Contacts were swabbed daily for up to 20 days, with a variety of tests performed on the swabs; relevant for this thesis is the RT-PCR test for SARS-CoV-2.
For all positive test results, the viral load was measured using the Ct value and then converted to a viral load using a standard curve.
Other data such as demography and symptoms was also collected.
For further details see \textcite{singanayagamDuration,hakkiOnset}.

\Textcite{hakkiOnset} analyse the viral load of the 57 contacts who tested positive for SARS-CoV-2 and had the growth phase in their viral load measured.
For 115 contacts with at least one positive RT-PCR test, the growth phase of viral load was not captured; these contacts were excluded from this analysis to ensure all individuals viral load trajectories could be identified.
More details on this analysis and my extensions to it can be found in \cref{E-ATACCC}.

\subsection{Coronavirus (COVID-19) Infection Survey} \label{intro:sec:cis}

\todo[inline]{Do I need to distinguish between estimates of positivity and prevalence?}

The CIS was setup in April 2020 to provide gold-standard measurements of the prevalence of SARS-CoV-2 in the community.
It is a longitudinal study with a representative sample of private households, this excludes settings such as care homes and student halls.
Initially, it was limited to England, but expanded to cover the whole of the UK in September 2020.
Enrolment was continuous until 31st Jan 2022, with data collected until 13th Mar 2023~\autocite{weiRisk}. 

The CIS had a household-based design, meaning that households were invited to participate in the study.
From households that participated, all individuals aged 2 and over were invited to participate.
Once invited, an enrolment swab would be taken at the first visit followed by 4 further weekly visits (giving a total of 5 swabs on days 0, 7, 14, 21, 28 relative to enrolment) after which visits were monthly.
As well as a swab to test for SARS-CoV-2, participants were asked to complete a questionnaire, and some participants were asked to provide blood samples.
Real-world issues meant that visits were often not on this precise schedule, and occasionally missed.
A full description of the study can be found in the study protocol~\autocite{cisProtocol}.

Up until July 2020, households were selected from those that had previously participated in an ONS survey~\autocite{CIStechData}.
Around 50\% of households agreed to participate, with around 90\% of eligible individuals (96,113 in total) in these households contributing multiple swabs to the survey.
After July 2020, addresses were also randomly sampled from a database of all UK addresses held by the ONS.
The response rate was lower, with only 12\% of households agreeing to participate, from which 86\% of eligible individuals (383,732 in total) contributed multiple swabs to the survey.

These rates of response raise the possibility of non-response bias.
In particular, some demographic groups, such as young adults and individuals of non-white ethnicity, were underrepresented among those who responded~\autocite{pouwelsCommunity}.
To correct for this, various methods have been used to adjust the estimates of prevalence.
I explain the preferred method, MRP (multilevel regression and post-stratification), in \cref{E-backcalc:sec:methods}.

In this thesis, I focus on the period from August 2020 until January 2021.
During the start of this period, the recruitment rate of CIS was expanded (see \cref{biology-data:fig:CIS-recruitment}(A)).
Following this, the recruitment then decreases and stabilises, as does the rate of swabs taken (see \cref{biology-data:fig:CIS-recruitment}(B)).
The recruitment pattern was chosen to hit targets regarding the fortnightly number of swabs.
Because the testing becomes less frequent after the first 4 weeks, some continuous recruitment is needed to maintain the number of swabs, as well as to replace participants who drop out.

\begin{figure}
  \centering \includegraphics{biology-data/CIS-recruitment}
  \caption[CIS swab numbers]{%
    Number of swabs taken by the CIS in the period considered in this thesis (Sep 2020 until Jan 2021).
    (A) Only enrolment (\ie first) swabs for each individual.
    (B) Total swabs.
    Note the differing y-axis scales.
    Data from \textcite{CIStechData}.
  }
  \label{biology-data:fig:CIS-recruitment}
\end{figure}


\begin{figure}
  \centering \includegraphics[width=\textwidth]{biology-data/CIS-positivity}
  \caption[CIS prevalence]{%
    CIS prevalence in the period considered in this thesis (Sep 2020 until Jan 2021).
    Dots show raw proportion of non-void tests that are positive on each day (days with fewer than 100 tests excluded for readability).
    Ribbon shows modelled 95\% CrI for the prevalence, corrected for non-response bias and smoothed over time (using the methodology described in \cref{E-backcalc:sec:methods}).
    The correction tends to increase the estimated prevalence because underrepresented groups tend to have higher prevalence.
  }
  \label{biology-data:fig:CIS-positivity}
\end{figure}
\begin{figure}
  \centering \includegraphics[width=\textwidth]{biology-data/CIS-num-tests}
  \caption[Number of CIS tests]{%
    Daily number of CIS tests conducted in the period considered in this thesis (Sep 2020 until Jan 2021).
  }
  \label{biology-data:fig:CIS-num-tests}
  \todo[inline]{Cut this repetition of number of tests? Maybe highlight period of interest in previous plot}
\end{figure}

\subsubsection{Episodes}

For the work in \cref{E-cis-perf-testing,E-cis-imperf-testing}, positive test results need to be grouped into \emph{infection episodes}.
An infection episode is a series of positive tests that are part of the same infection.
I use a method developed throughout the pandemic, and published within \textcite{weiRisk}; see that publication for full justification of the criteria.
Denote the time of a positive test by $t$, the number of consecutive negative tests prior to $t$ by $n$, and the time of the first positive test in the same individual by $t_p$ ($t_p = -\infty$ if there is no previous positive test in the same individual).
The test at $t$, in the pre-Omicron era (prior to December 2021), is defined as the start of a new infection episode if any of the following apply.
\begin{enumerate}
  \item $t_p = -\infty$.
  \item $n \geq 1$ and $t - t_p > 120$.
  \item $n \geq 2$ and $t - t_p > 90$.
  \item $n \geq 3$ and $t - t_p > 60$.
  \item $n \geq 4$.
  \item A complex heuristic based on the likely variant of the positive tests (see \textcite{weiRisk} for details). This heuristic rarely applied in the pre-Omicron era~\citePersonalComms{Sarah Walker}.
\end{enumerate}

A negative between two positive tests, when those positives are classed as part of the same infection episode, is known as an \emph{intermittent negative}.
Intermittent negatives are the clearest example of a false negative.
An intermittent negative is when an individual tests negative but tested positive previously and subsequently.
Intermittent negatives are stripped out when creating the dataset used for all duration analyses in this chapter, but demonstrate that false negatives do occur.

% Whether a future positive is part of the same episode of a reinfection is not always trivial; I rely on a process developed previously by Sarah Walker, see \cref{E-episode-def}.


\ifSubfilesClassLoaded{
  \listoftodos
}{}

\end{document}