\documentclass[thesis.tex]{subfiles}

\begin{document}
\ifSubfilesClassLoaded{
    \setcounter{chapter}{6}
}

\chapter{Estimating transmission} \label{transmission}

In this chapter, I use the estimate of the distribution of the duration of RT-PCR positivity based on combining information from the ATACCC and CIS studies (see \cref{E-imperf-test:sec:conclusion}) to investigate the feasibility of estimating incidence and transmission of SARS-CoV-2 from only the CIS data (described in \cref{E-intro:sec:cis}).
I will show that the CIS data are sufficient, demonstrating the value of surveys of this type for pandemic surveillance.

I take two approaches, one phenomenological and one mechanistic, both previously discussed in \cref{E-inc-prev:sec:infection-process}.

I start with the phenomenological approach (\cref{backcalc}), based on \cref{E-inc-prev:eq:EPt}, which enables estimation of incidence within (sub)populations of England.
The approach makes few assumptions about how transmission occurs, decreasing the potential to introduce bias and can also easily incorporate arbitrary covariates to adjust for non-response bias.
The scale of the CIS and desire to correct for non-response bias presents challenges for this methodology, especially within the limited computation of the SRS environment.

In \cref{SEIR}, I use a mechanistic compartmental model, within the modelling framework introduced in \cref{E-SEIR:sec:mechanistic-models}.
I estimate both incidence and several components of transmission.
Estimating transmission from CIS data requires developing a novel observation model.

Finally, in \cref{transmission:sec:comparison}, I compare the results of the two approaches.
I show they give similar values, providing reassurance of the robustness of the results to modelling assumptions.
% Overall, there is a bias-variance trade-off between the models.
% Incidence and transmission have not previously been estimated from only SARS-CoV-2 prevalence surveys before.

\section{Phenomenological approach} \label{backcalc}

In this section, I apply the theory introduced in \cref{E-inc-prev} to estimate incidence.
I view the CIS as a series of daily, cross-sectional prevalence surveys, as previously used to estimate prevalence (see \cref{E-biology-data:sec:MRP}).

An important consideration is non-response bias in the CIS study.
I correct for the non-response through the poststratification method introduced in \cref{E-biology-data:sec:MRP}.
Poststratification requires estimates of incidence in each stratum, before combining them to the target population and subpopulations of interest.
The strata are defined by at least age group, region, ethnicity, and sex due to differential non-response in the survey (see \cref{E-intro:sec:cis}).

\subsection{Approaches} \label{backcalc:sec:approach}

In \cref{E-inc-prev:sec:observation-process}, I showed that a deterministic approximation to the relationship between true prevalence and incidence is a good approximation for estimating incidence.
This means that once one of the two quantities is inferred, the other can be calculated via a deterministic function.
This gives two possible approaches for estimating incidence from prevalence data; in \cref{transmission:sec:backcalc-properties} I argue the second approach is more appropriate in this context.

% \subsubsection{Possible approaches}

The first approach, which I call the \emph{incidence-inferred} approach, specifies a single statistical model including both incidence and prevalence.
The basis of this statistical model, in a Bayesian framework, would be as follows:
\begin{align}
    \vec{Z} &\dist F_Z \\
    P_{i,t}  &= \sum_{k=0}^{d_\text{max}-1} Z_{i,t-k} S(k+1) \\
    \rho &\dist F_\rho \\
    Y_{i,t} \mid \vec{P}, \rho &\dist \BB(n_{i,t}, P_{i,t}, \rho)
\end{align}
where $\vec{Z}$ is a vector such that $Z_{i,t}$ is the incidence proportion in stratum $i$ on day $t$, with a joint prior $F_Z$; $P_{i,t}$ is the prevalence in stratum $i$ on day $t$; $\rho$ is the overdispersion parameter of the beta-binomial distribution (see \cref{E-distributions}) with prior $F_\rho$; $Y_{i,t}$ is the number of positive tests in stratum $i$ on day $t$; and $n_{i,t}$ is the number of tests in stratum $i$ on day $t$.
Fitting this model to a prevalence survey such as the CIS gives a posterior distribution of $\vec{Z}$, the target of inference.
The relationship between $\vec{P}$ and $\vec{Z}$ is from \cref{E-inc-prev:eq:EPt}, although dividing both sides by the stratum's population (assumed constant) to have the values in terms of incidence proportion and prevalence rather than incidence and number of prevalent individuals.

Specifying $F_Z$ should consider the need to provide structure to stabilise the problem (see \cref{E-intro:sec:disease-model}).
In the context of the CIS, it needs to allow borrowing information across time and/or strata (see \cref{E-biology-data:sec:MRP}), which is required because for some stratum/time combinations the data become very sparse.

The second approach, which I call the \emph{prevalence-inferred} approach takes a two-step approach.
First, it estimates prevalence using the statistical model from \cref{E-biology-data:sec:MRP}.
Then, it transforms the posterior distribution of prevalence to a posterior distribution of incidence using the forward substitution as in \cref{inc-prev:eq:forward-substitute}.
At a high-level, the statistical model is:
\begin{align}
    \vec{P}  &\dist F_P \\
    \rho &\dist F_\rho \\
    Y_{i,t} \mid \vec{P}, \rho &\dist \BB(n_{i,t}, P_{i,t}, \rho)
\end{align}
where $F_P$ is analogous to $F_Z$ in providing a joint prior for the prevalence in each stratum and day combination, and all other notation is as in the incidence-inferred approach.
The first step in the approach is to calculate the posterior distribution for $\vec{P}$.

The second step is to transform the posterior distribution for $\vec{P}$ to a posterior distribution for $\vec{Z}$.
The transformation is based on \cref{inc-prev:eq:forward-substitute}, which gives a one-to-one mapping between $\vec{P}$ and $\vec{Z}$; implementation details are in \cref{backcalc:sec:methods}.

\subsection{Desirable properties} \label{transmission:sec:backcalc-properties}

The incidence-inferred approach is more parsimonious in being a single step procedure.
However, computational challenges make the prevalence-inferred approach more appealing.

It will be convenient to define a variable equating to the quantity initially estimated: let $\pi_{i,t} = Z_{i,t}$ under the inference-inferred approach and $\pi_{i,t} = P_{i,t}$ under the prevalence-inferred approach.

The chosen approach should have the following properties.
\begin{enumerate}
    \item The prior for $\pi_{i,t}$ should favour $\pi_{i,t}$ changing exponentially in $t$, \ie penalize changes from exponential growth.
        % Penalizing changes from continuous exponential growth is a natural formulation because mechanistic models predict exponential growth over short periods of time.
        Such penalization has previously been shown to improve estimates of SARS-CoV-2 prevalence by balancing over and under-fitting~\autocite{ealesAppropriately} and is the approach used for the CIS prevalence model (see \cref{E-biology-data:sec:MRP}).
    \item Limit estimates to epidemiologically and biologically plausible values.
        That is, incidence proportion and prevalence should be in the interval $[0, 1]$.
        % While incidence proportion can theoretically be greater than 1, this would imply that some individuals were infected multiple times in a day which is not biologically plausible.
    \item Include sufficient parameters to incorporate all necessary covariates to correct for non-response bias.
    \item Borrow information across the different covariates.
        The borrowing should penalize changes in the distribution of incidence/prevalence across strata in the same geographic region.
        A constant distribution is predicted by the theoretical study of epidemics, for example the dynamics of stratified, mechanistic transmission models (such as the one introduced in \cref{E-SEIR:sec:structured-populations}) when behavioural and immunity changes are negligible.
    \item Be computationally efficient.
        The model will be fitted to granular data and therefore needs to be run within the SRS, with its computational limitations (see \cref{E-biology-data:sec:SRS}).
\end{enumerate}

A parsimonious model satisfying properties 1 and 2 is to specify $\logit(\pi_{i,t}) = g_{\vec{\varphi}}(i, t)$, where $g$ is a function parameterised by ${\vec{\varphi}}$ and the prior on ${\vec{\varphi}}$ is specified to favour $d^2g/dt^2 = 0$, \ie favour linear growth in $\logit(\pi_{i,t})$.
The prior $F_Z$ or $F_P$ is now a prior on ${\vec{\varphi}}$.
The $\logit$ link function both limits $\pi_{i,t}$ to the interval $[0, 1]$, and logit-linear growth approximates exponential growth for small $\pi_{i,t}$~\autocite{ealesAppropriately}.

The logit transformation also allows property 4 to be conveniently satisfied using an additive model for $g_{\vec{\varphi}}$.
An additive model on the logit scale is approximately multiplicative on the natural scale for small values of $\pi_{i,t}$, which will be common.

This form of the model includes the MRP prevalence model in \cref{E-biology-data:sec:MRP}.

% Combined with property 4, the form of $g$ the null model is $g_\varphi(i, t) = \varphi_0 + \varphi_1 t + f(i)$ where $a_0$ and $a_t$ are model parameters and $f$ is a function to be specified; that is, changes away from this model should be penalized.

% Properties 3 and 4 suggest the extension of the previously-developed MRP method, used to already estimate prevalence, which already satisfies these properties.
% This method has previously been used successfully to estimate prevalence, corrected for non-response bias, from the CIS data~\autocite{cisMethodsONS,pouwelsCommunity,pouwelsMRPvaccination}.


Properties 3 and 5 suggest the use of an approximate method for computing the posterior distribution.
A particular issue is that using MCMC, sufficient covariates to adjust for the non-response bias, and a period of more than 6--8 weeks leads to a model that is computationally prohibitive ~\citePersonalComms{Koen Pouwels}.
Instead, previous MRP implementations for the CIS used INLA (see \cref{E-biology-data:sec:MRP}).

INLA-based inference can only be applied to LGMs (\emph{Latent Gaussian Models}); see \cref{E-transmission:sec:INLA} for further details on INLA and LGMs.
The LGM framework requires that a transformation of $P_{i,t}$ is linear in $\vec{g} = [g_{{\vec{\varphi}}}(1, 1), \dots, g_{{\vec{\varphi}}}(1, T), \dots, g_{{\vec{\varphi}}}(2, 1), \dots, g_{{\vec{\varphi}}}(I, T)$ where $I$ and $T$ are the number of strata and days respectively.

If $\pi_{i,t} = P_{i,t}$ then this is trivially satisfied.
However, if $\pi_{i,t} = Z_{i,t}$, then no such linear relationship exists.
In particular:
\begin{align}
    P_{i,t}
    &= \sum_{k=0}^{\dmax} \pi_{i,t-k} S(k+1) \\
    &= \sum_{k=0}^{\dmax} \expit(g_{\vec{\varphi}}(i, t-k)) S(k+1).
\end{align}

Therefore, satisfying all the desirable properties requires the prevalence-inferred approach, which I adopt.

\subsection{Methods} \label{backcalc:sec:methods}

Now I explain how I extended the MRP model introduced in \cref{E-biology-data:sec:MRP} to implement the prevalence-inferred approach.
I start with the prevalence posterior from \cref{E-biology-data:sec:positivity-results}, denoting the times the estimates are for as $t = 1, \dots, T$.
I draw 500 samples of $\vec{P}$ from the approximate posterior distribution (using the procedure described in \cref{transmission:sec:INLA:posterior}).

Then, I transform this posterior sample using \cref{transmission:alg:calculate-Z}.
The algorithm is based on the forward-substitution algorithm from \cref{inc-prev:eq:forward-substitute}, which including stratum is:
\begin{align}
Z_{i,t}
&= P_{i,t} - \sum_{j=1}^{t-1} S(t - j + 1) Z_{i,j}.
\label{transmission:eq:forward-substitute}
\end{align}
Dividing each side of \cref{inc-prev:eq:forward-substitute} by the total number of individuals in each stratum transforms the number of prevalent individuals to prevalence and incidence to incidence proportion.
The algorithm uses a deterministic relationship between incidence and prevalence within a stratum, as \cref{E-inc-prev:sec:observation-process} justifies; \ie, the estimate of the incidence is negligibly affected by approximating the distribution $\vec{P} \mid \vec{Z}$ by its mean.

\begin{algorithm}
    \For{$j = 1, \dots, 500$}{
      Set $\vec{P}^{(j)}$ to a posterior sample of $\vec{P}$ \;
      \For{each stratum $i$}{
        \For{$t = -\dmax, \dots, 0$}{
            $Z^{(j)}_{i,t} = P^{(j)}_{i,1} / \sum_{k=1}^{\dmax} S(k)$
        }
        \For{$t = 1, \dots, T$}{
            Set $Z^{(j)}_{i,t} = P^{(j)}_{i,t} - \sum_{k=-\dmax}^{t-1} S(t - k + 1) Z^{(j)}_{i,k}$ \;
        }
      }
    }
    \caption{%
      Algorithm to transform 500 posterior samples posterior samples of $\vec{P}$ to posterior samples of $\vec{Z}$.
      Assumes constant incidence for $t \leq 1$ where $t = 1$ is the first day of prevalence data.
      Calculation of $Z^{(j)}_{i,t}$ is from \cref{transmission:eq:forward-substitute}.
    }
    \label{transmission:alg:calculate-Z}
\end{algorithm}

\Cref{transmission:eq:forward-substitute} is derived from \cref{inc-prev:eq:EPt-matrix}: $\vec{P} = \matr{S} \vec{I}$.
For this to have a unique solution, $\vec{I}$ and $\vec{P}$ need the same dimension.
However, I am using a period of prevalence which starts after the first infections.
These prior infections will still contribute to the prevalence.
I assume that the incidence for $t \leq 1$ is constant.
Therefore, $P_1 = I_0 \sum_{i=1}^{\dmax} S(i)$ and $I_t = P_1 / \sum_{i=1}^{\dmax} S(i)$ for $t \leq 1$.

% First, I draw a sample of 500 prevalence time series in each stratum from the approximate posterior distribution (using the procedure described in \cref{transmission:sec:INLA:posterior}).
% For each sample and stratum, I use the relationship in \cref{E-inc-prev:eq:EPt} to calculate the incidence proportion at each time point.
% I consider the relationship between incidence and prevalence within a stratum to be deterministic, as \cref{E-inc-prev:sec:observation-process} justifies, that is the estimate of the incidence is negligibly affected by approximating the distribution $\vec{P} \mid \vec{Z}$ by its mean.
% I solve this system using forward substitution, calculating the value of $Z_t$ in \cref{inc-prev:eq:forward-substitute} sequentially for $t = 1, \dots, T$.
% $t = 1$ is defined as the first day where prevalence data are used.
% \Cref{inc-prev:eq:forward-substitute} is repeated here for convenience:
% \begin{align}
% Z_t
% &= \frac{P_t - \sum_{i=1}^{t-1} S_{ti} Z_i}{S_{tt}} \\
% &= P_t - \sum_{i=1}^{t-1} S(t - i + 1) Z_i.
% \label{transmission:eq:forward-substitute}
% \end{align}
% % \Cref{E-inc-prev:eq:EPt}, considered across different values of $t$, can be written as $\vec{P} = \matr{S} \vec{Z}$, where: $\vec{P}$ is a vector of the number of prevalent individuals at each time point in the stratum; $\matr{S}$ is a matrix formed from the survival function, as defined below; and $\vec{Z}$ is a vector of the incidence at each time point in the stratum.
% Dividing each side by the total number of individuals in each stratum transforms the number of prevalent individuals to prevalence and incidence to incidence proportion.

Violation of assumptions in various parts of this procedure can lead to the second term in \cref{transmission:eq:forward-substitute} to be greater than the first, giving negative incidence.
In this case, the incidence prior to day $t$ imply that the prevalence on day $t$ is higher than that predicted by the fitted regression model for this strata/day combination.

For each posterior sample of $\vec{Z}$, I poststratify it to the CIS target population and subpopulations of interest, using the same basic method as in \cref{E-biology-data:sec:MRP}.
Specifically, define a (sub)population of interest as a set of strata $\set{S}$.
Then the incidence proportion in the (sub)population on day $t$ is
\begin{align}
Z_t(\set{S})
&= \frac{\sum_{j \in \set{S}} Z_{j,t} N_j}{\sum_{j \in \set{S}} N_j}
\label{backcalc:eq:poststratify}
\end{align}
where $N_j$ is the population size of stratum $j$.
The poststratified samples are now from the posterior of interest.

The posterior samples of $\vec{P}$ are produced within the SRS and then extracted.
I implement the calculation of $\vec{Z}$ and poststratification in R~4.2.0~\autocite{R-4-2-0} using the built in \texttt{forwardsolve} function, tidybayes~3.0.6~\autocite{tidybayes}, and the tidyverse set of packages~\autocite{tidyverse}.

\subsection{Results and discussion} \label{backcalc:sec:results}

A slow increase in incidence is estimated to occur over the autumn 2020 period, peaking in October in most regions (see \cref{transmission:fig:backcalc-regions}).
An exception is the North East and East of England regions (see \cref{transmission:fig:backcalc-regions}(C) and (D) respectively); these regions show no consistent increase or decrease in incidence during October.
All regions then decrease during November; partially coincidng with the national lockdown in force in England between 5 Nov and 2 Dec 2020~\autocite{ifgTimeline}.

During November, incidence starts rising, with regional heterogeneity in the timing of when this occurs (see \cref{transmission:fig:backcalc-regions}).
The most explored explanation for this is the emergence of the Alpha variant, which was more transmissible than previous variants and emerged at different times across the country~\autocite{walkerTracking,daviesEstimated,lythgoeLineage}.
There is weak evidence that when incidence starts increasing is correlated with the timing of the Alpha variant's spread (see \cref{transmission:fig:backcalc-alpha}).
A second peak occurs in late December or early January in all regions; this second peak is larger than the first in all regions except the Northern three (North East, North West, and Yorkshire and the Humber).
\begin{figure}
    \includegraphics{transmission/backcalc-regions}
    \caption[Incidence estimated using the phenomenological approach by region]{%
        Incidence proportion for (A) all of England and (B) its component regions.
    }
    \label{transmission:fig:backcalc-regions}
\end{figure}
\begin{figure}
    \includegraphics{transmission/backcalc-alpha}
    \caption[Correlation of the emergence of Alpha and minimum incidence.]{%
        x-axis: time that S-gene target failure, a marker of the Alpha variant, is estimated to start increasing from baseline by region among CIS test results; from \textcite[table S3]{walkerTracking}.
        y-axis: posterior median and 95\% CrI of the date of minimum incidence in November or early December in each region.
        Linear regression line shown.
    }
    \label{transmission:fig:backcalc-alpha}
\end{figure}

The incidence proportion was highest among the 12--16 and 17--25 age groups throughout the period (see \cref{transmission:fig:backcalc-ages}).
Except the 17--25 age group, all age groups show similar trends (\ie parallel lines on a log scale), meaning there was insufficient information in the data to overcome the prior which favoured this situation.
The incidence proportion in the 17--25 age group decreases faster than other age groups during November.
There are many plausible explanations for this effect.
The higher incidence in this age group prior to November means it could be an immunity effect.
Alternatively, it could be a consequence of schools remaining open in the November lockdown; 20--24 year olds rarely interact with school-aged children (see \cref{E-SEIR:fig:age-contacts})
Many other explanations are possible, and further work is required to determine causality.
\begin{figure}
    \makebox[\textwidth][c]{\includegraphics[width=.9\paperwidth]{transmission/backcalc-ages}}
    \caption[Incidence estimated using the phenomenological approach by age group]{%
        Incidence proportion for each age group, on both natural and log scales.
    }
    \label{transmission:fig:backcalc-ages}
\end{figure}

An edge effect, a rapid rise in incidence between days one and two, is visible (see \cref{transmission:fig:backcalc-start-effect}).
This is due to the incorrect assumption that incidence is constant in the past.
Changing this assumption (\eg to exponential growth) modifies the estimated value on the first day, but the rest of the series is insensitive to this change (not shown).
Therefore, the first day's estimate is unreliable, and has been excluded from all other results.
However, the rest of the series is robust to this assumption.
\begin{figure}
    \centering \includegraphics{transmission/backcalc-start-effect}
    \caption[Edge effects in the phenomenological approach method]{%
        First seven days of incidence proportion estimate for the whole of England, highlighting the edge effect.
    }
    \label{transmission:fig:backcalc-start-effect}
\end{figure}

The issues with negative incidence are likely caused by issues with modelling assumptions.
There are two sets of assumptions which are most likely to be violated.

The first set of assumptions is in estimating the survival function, $S(t)$, which is assumed to be known with no uncertainty in this procedure.
However, \cref{E-imperf-test} showed that the estimate of $S(t)$ is sensitive to assumptions made, such as the assumed $\psens$ and strength of priors used.

The second set of assumptions are in specifying the prior for $\vec{P}$.
In particular, the prior places non-zero mass on prevalence values which imply negative incidence under the other model assumptions.
Rectifying this assumption is challenging without switching to the inference-inferred approach.
The inference-inferred approach could also allow more interpretable priors to be used because the infection process is the process that corresponds to biological and behavioural processes.
The inference-inferred approach, and other more powerful modelling approaches, require an alternative inference technique allowing a broader set of models than LGMs.

Various alternative inference techniques could be considered.
If the infection process is considered a Poisson process (defined in \cref{E-inc-prev:sec:poisson-process}), then, conditional on the process's intensity, there is independence between individuals' infection status.
The conditional independence implies the incidence proportion and prevalence are the mean of many independent Bernoulli variables.
Therefore, the Central Limit Theorem implies they are approximately normally distributed.
Exploiting this property could enable efficient approximations to the posterior.
For example, a Laplace approximation or variational inference~\autocite{bleiVariational}.
These methods require further work to check the quality of the approximations in this context.


\section{Mechanistic modelling} \label{SEIR}

In this section, I apply the general mechanistic transmission modelling framework from \cref{E-SEIR} to the CIS prevalence data.
The application of this framework follows \textcite{birrellRealtime}.
The largest modification from the generic framework presented in \cref{E-SEIR} is the parameterization of the WAIFW matrix (\cref{SEIR:sec:WAIFW-application}).
A novel approach (\cref{SEIR:sec:observation}) links this transmission model to the CIS prevalence data; the approach is generalisable to other studies collecting representative prevalence data, \eg REACT.
In the mechanistic modelling literature, this part of the model is referred to as the observation model, however, in the terminology of \cref{E-inc-prev}, it includes both the prevalence process and the observation process.
Having specified the full model (\cref{SEIR:sec:methods-application}), I conducted a simulation study to validate the inference process and identifiability (\cref{SEIR:sec:sim-study}), before applying the model to the CIS data (\cref{SEIR:sec:application}).
In \cref{SEIR:sec:discussion}, I discuss the results; in \cref{transmission:sec:comparison} I compare the results to those in the previous section.

In common with \cref{E-SEIR:sec:mechanistic-models}, the notational conventions in this section differs slightly from the rest of the thesis.

\subsection{Methods} \label{SEIR:sec:methods-application}

% The model will be stratified only by region and age group, so the data are aggregated to these levels.
% Only individuals aged two or older are eligible to participate in the CIS; I assume that test results in all participants aged under 11 are still representative of the whole 0--11 age group.

I model each region fully independently, \ie with no shared parameters or transmission between regions.
As well as the motivation given in \cref{E-backcalc}, independent regions allow heterogeneity in the parameters between regions, which could arise from a range of socioeconomic factors.
Furthermore, the assumption of homogeneous mixing within age groups is more plausible for regions than the whole of England.
By modelling each region independently, I assume that the epidemics in each region do not interact.
Clearly, this is not realistic as travel between regions of England is common and unrestricted.
However, except in an epidemic's early stages, the interaction between large regions has a negligible impact on the epidemic's progression~\autocite[e.g.][]{birrellRealtimea,gogSpatial,eggoSpatial}.

\subsubsection{WAIFW matrix} \label{SEIR:sec:WAIFW-application}

A parsimonious model is needed to parameterize the WAIFW matrix.
In \cref{E-SEIR:sec:structured-populations}, I explained how the elements of the WAIFW matrix, $\matr{\beta}$ can be decomposed as $\beta_{aa'} = \kappa_{aa'} q_{aa'} N_{a'} / N_{a}$ where\todo{should I repeat these definitions? they already exist in ch 3}: $\kappa_{aa'}$ is the mean rate at which an individual in stratum $a'$ makes contact with individuals in stratum $a$; $q_{aa'}$ is the probability of transmission upon contact between an infectious individual in stratum $a'$ and a susceptible individual in stratum $a$; and $N_{a}$ is the total number of individuals in stratum $a$.
For many epidemics, where the population behaviour is fairly constant over time, a constant $\matr{\beta}$ is a good approximation.
However, in SARS-CoV-2, population behaviour changed dramatically, both spontaneously (\eg due to fear) and in response to government-imposed restrictions.
Therefore, following \textcite{birrellRealtime}, I introduce a time-dependence giving: $\beta_{aa'}(t) = \kappa_{aa'}(t) q_{aa'}(t) N_{a'} / N_a$; $N_a$ and $N_{a'}$ are constant because the change in the total population is negligible over the length of time considered here.

I take information on the contact rates, $\kappa_{aa'}(t)$, from several data sources, using the methodology of \textcite{vanleeuwenTime,vanleeuwenAugmenting}.
%; these estimates have been used for previous modelling of the SARS-CoV-2 pandemic~\autocite{vanleeuwenTime,birrellRealtime}.
% These start with estimates of contact in each setting, based on \textcite{vanleeuwenAugmenting}.
This method starts with pre-pandemic baseline estimates of $\kappa_{aa'}$ from the POLYMOD study~\autocite{mossongSocial}, and time use from the UKTUS (UK Time Use Survey)~\autocite{UKTUS}.
POLYMOD asked a representative sample of the UK population about the contacts they make each day, including information on each contact's age and location (home, work, school, leisure, transport, or other).
UKTUS asked a representative sample of the UK population about how much time they spent doing various activities (\eg shopping and spending time at home) each day.
\Textcite{vanleeuwenAugmenting} combined these two datasets to estimate the rate of contacts between age groups during each activity.
\Textcite{birrellRealtime} then modify these contact rates based on measured changes in time use during the pandemic.
These changes are measured using Google mobility data~\autocite{googleCOVID19} and school attendance data~(unpublished).
Google mobility data are based on the location of Android users over time, relative to a pre-pandemic baseline.
The mobility and school data are averaged over a week down or up-weight the number of contacts in each setting, before combining contacts across settings to form $\kappa_{aa'}(t)$.
$\kappa_{aa'}(t)$ is assumed to be piecewise constant, changing only at the start of each week.
For further details see \textcite{vanleeuwenAugmenting} and \textcite[supplementary material]{birrellRealtime}.

The probability of transmission, $q_{aa'}(t)$, needs further decomposition to be identifiable, for the same issue of dimensionality preventing each element of $\matr{\beta}$ being estimated independently (see \cref{E-SEIR:sec:structured-populations} for a full explanation).
I assume that $q_{aa'}(t) = \beta(t) q^\text{susc}_{a} q^\text{inf}_{a'}$ where $\beta(t)$ is a time-varying reference probability of transmission; $q^\text{susc}_a$ is the relative \emph{susceptibility} of stratum $a$, \ie how vulnerable to infection individuals in stratum $a$ are; and $q^\text{inf}_{a'}$ is the relative \emph{transmissibility} of stratum $a'$, \ie how likely individuals in stratum $a$ are to pass on an infection.
These measures include both biological differences, \eg due to differences in immune systems, or behavioural, \eg if infants are less hygienic than adults.

The time-varying component, $\beta(t)$, allows for behavioural changes that impact transmission but are not captured by the contact matrices (\eg mask wearing).
I impose a random walk prior on $\log\beta(t)$ on the weekly timescale, smoothing its value over time.
This prior encodes that behaviour changes smoothly over time.
\begin{align}
    \beta(t) &= \beta_{\tilde{w}(t)} \\
    \log\beta_{w+1} &= \log\beta_w + \sigma_\epsilon \epsilon_w \\
    \epsilon_w &\sim \N(0, 1)
\end{align}
where $\tilde{w}(t)$ is the week number of time $t$, with week 1 being the first week considered.
Two hyperparameters will be inferred: $\beta_1$, the initial value of the random walk; and $\sigma_\epsilon$, the standard deviation of the random walk (see \cref{SEIR:sec:priors}).
$\beta(t)$ is the only time-varying fitted parameter.
Therefore, in practice, it will also absorb other effects such as any error in $\matr{\kappa}$.

% $K_{ij}$ and $\beta_{ij}$ appear only as a product and therefore are not individually identifiable.
% Normally, $K_{ij}$ is assumed to be known, for example from a previous contact survey~\autocite[such as][]{mossongSocial}.
% Ideally, all the $\beta_{ij}$ terms would be estimated, but there is normally inadequate data to do so, requiring assumptions to be made to reduce the number of parameters~\autocite[65]{keelingModeling}.
% % $\beta_{ij}$ is then estimated, although often with a parsimonious model.
% A common assumption is that $\beta_{ij}$ can be decomposed into two independent components such that $\beta_{ij} = \beta \beta^\text{susc}_{i} \beta^\text{inf}_j$.
% For any stratum $i$, $\beta^\text{susc}_i$ is known as the \emph{relative susceptibility} of stratum $i$.

%Now, we assume that $\beta_{ij} = \beta(t) \beta_{ij}$ where $\beta(t)$ is a non-strata-specific, time-varying component representing precautions taken by the population, for example masking; and $\beta_{ij}$ represents a constant probability of transmission from an infectious individual in strata $j$ to a susceptible individual in strata $i$ given contact, which could, for instance, be affected by differential susceptibility to infection for different age-groups.
%We arbitrarily choose one $\beta_{ij}$ to be equal to 1 for identifiability and hence the other $\beta_{ij}$ values are relative to this reference class.

Studies of pre-Alpha SARS-CoV-2 suggest that all individuals have similar infectiousness but that pre-adolescent children are less susceptible than adults~\autocite{chenRole,vinerTransmission}.
Therefore, I set $q^\text{inf}_{a'} = 1$ for all $a'$, $q^\text{susc}_{a} = q_c$ for 0--11 year olds, and $q^\text{susc}_{a} = 1$ for the remaining strata.
$q_c$ can be interpreted as the relative probability that a child is infected (compared to an adult), given contact with an infectious individual.
%$q_c$ is then estimated, although using an informative prior (see \cref{SEIR:sec:priors}).

\subsubsection{Prevalence and observation models} \label{SEIR:sec:observation}

This section  explains the novel observation model approach I propose for fitting to the CIS prevalence data.
This will link the model in the previous section to the CIS prevalence data.

% Here, I extend the system of ODEs to include compartments representing RT-PCR positive individuals.
% The proportion of RT-PCR positive individuals in each strata is then linked to the CIS using a Beta-Binomial likelihood, with the modelled proportion that are RT-PCR positive as the mean of the Beta distribution.
One approach would be to convolve the incidence based on the transmission model with the RT-PCR duration distribution.
Here, the daily incidence proportion for each stratum derived by solving the system of ODEs governing the transmission model (see \cref{E-SEIR:sec:SIR}), and then prevalence calculated using \cref{E-inc-prev:eq:EPt}.
However, this approach is computationally expensive, likely prohibitively expensive if performed within a MCMC loop.

A computationally cheaper option, which I will take, is to approximate this convolution by adding compartments to the ODEs.
This approach allows an infected, pre-positive compartment, which I denote $p_0$, to be included.
Up until now, this stage of the disease (shown in \cref{E-biology-data:fig:natural-history}) has been neglected, assumed to be of negligible duration.
An advantage of this modelling approach is that it can be included without complications (\eg issues of numerical stability).

$p_0$ increases at the same rate as individuals flow into $e_1$.
These individuals then transition into a set of compartments, $p_1, \dots, p_l$, each with their own transition rate.
I will choose $l$ and the transition rates such that combined length of stay (\ie time from entry in $p_1$ until exit from $p_l$) approximates the duration of detectability distribution estimated in \cref{E-imperf-test}.
This idea extends the technique of having multiple exposed or latent compartments (introduced in \cref{E-SEIR:sec:non-exponential}).
While the use of exponential or Erlang (gamma distributions with integer first parameter) is common in the observation model of compartmental models~\autocite[e.g.][]{overtonEpiBeds}, the use of more flexible phase-type distributions, as here, has rarely, if ever, been used in practice~\autocite{hurtadoGLCT}.
This broader class of distributions allows the model to capture the tail of the distribution, which is important when incidence suddenly decreases.

\Textcite{osogamiClosed} provide a method to construct the compartments, implemented in the mapfit R package~\autocite{mapfit}. 
The resulting compartmental approximation matches the original distribution up to the first three moments.
Applying the package to the posterior mean of the distribution estimated in \cref{E-imperf-test} gives the model in \cref{SEIR:fig:full-model} with the parameter estimates in \cref{SEIR:table:ec-params} (see \cref{transmission:sec:phase-type} for details).
\begin{table}
    \centering
    \begin{tabular}{c c c c c}
        $l$ & $\omega_1$ & $\omega_{2}$ & $\omega_{3}$ & $\chi$ \\
        3 & 0.0820 & 0.126 & 0.0223 & 0.00973  \\
    \end{tabular}
    \caption{Parameter estimates for the phase-type distribution with the same first three moments as the posterior mean of the duration distribution estimated in \cref{E-imperf-test}.}
    \label{SEIR:table:ec-params}
\end{table}
% end table of ec params

I link the modelled proportion of individuals that are RT-PCR positive to the data using a beta-binomial likelihood (as discussed in \cref{E-biology-data:sec:clustering}).
If on each day $t$, each stratum $i$ is observed to have $y_{at}$ positives out of $n_{at}$ tests, then the likelihood is:
\begin{align}
    \prod_{a,t} \BB (y_{at} \mid n_{at}, \bar{p}_{at}, \rho)
    \label{SEIR:eq:likelihood}
\end{align}
where: $\bar{p}_{at}$ is the mean proportion of individuals in strata $a$ that are RT-PCR positive on day $t$; and $\rho$ is a nuisance parameter controlling the overdispersion of the beta-binomial distribution.
Denoting the proportion of stratum $a$ in state $p_k$ as $p_{ka}(\tau)$, then $\bar{p}_{at} = \int_{t}^{t+1} \sum_{k=1}^l p_{ka}(\tau) d\tau$.

\subsubsection{Model equations and solution} \label{SEIR:sec:full-model}

\begin{figure}
\makebox[\textwidth][c]{
\begin{tikzpicture}[
    node distance = 2.5cm,
    on grid,
    auto,
    ->,>=stealth',
    every state/.style={draw,rectangle},
    ]

    \node[state] (S) {$s$};
    \node[state, right=of S] (E1) {$e_1$};
    \node[state, right=of E1] (E2) {$e_2$};
    \node[state, right=of E2] (I1) {$i_1$};
    \node[state, right=of I1] (I2) {$i_2$};
    \node[state, right=of I2] (R) {$r$};

    \path (S) edge node {$\lambda$} (E1)
          (E1) edge node {$\sigma$} (E2)
          (E2) edge node {$\sigma$} (I1)
          (I1) edge node {$\gamma$} (I2)
          (I2) edge node {$\gamma$} (R);

    \node[state, below=3cm of E1] (P0) {$p_0$};
    \node[state, right=of P0] (P1) {$p_1$};
    \node[state, right=of P1] (P2) {$p_2$};
    \node[state, right=of P2] (P3) {$p_3$};
    \node[state, right=of P3, draw=none] (P4) {};

    \path  (S) edge node {$\lambda$} (P0)
          (P0) edge node {$\omega_0$} (P1)
          (P1) edge node {$\omega_1$} (P2)
          (P2) edge node {$\chi \omega_{2}$} (P3)
               edge [out=-45,in=-135] node[below] {$(1 - \chi) \omega_{2}$} (P4)
          (P3) edge node {$\omega_{3}$} (P4);
    
    \node[draw, thick, inner sep=0.3cm, fit=(P1) (P3), fill=blue!10,opacity=0.2] {};
\end{tikzpicture}
}
  \caption[SEIR model]{SEIR model used for the analysis in this section. The shaded box represents RT-PCR-positive individuals. Shown for a single stratum.}
  \label{SEIR:fig:full-model}
\end{figure}

The remaining details of the transmission model structure follow \textcite{birrellRealtime}.
The transmission model is an SEIR model with two latent and two infectious states.
To form whole of England estimates, each region is modelled independently with the results aggregated using the poststratification methodology explained in \cref{backcalc:sec:methods}.

A diagram of the model for a single stratum is shown in figure \cref{SEIR:fig:full-model}.
The full system of differential equations for each region is below.
\begin{equation}
\begin{aligned}
    \vec{\lambda}(t) &= (\matr{\kappa}(t) \circ \matr{q}(t) \circ \matr{N}) ((\vec{i}_1(t) + \vec{i}_2(t))) \\
    \frac{d\vec{s}(t)}{dt} &= -\vec{\lambda}(t) \circ \vec{s}(t) \\
    \frac{d\vec{e}_1(t)}{dt} &= \vec{\lambda}(t) \circ \vec{s}(t) - 2\sigma \vec{e}_1(t) \\
    \frac{d\vec{e}_2(t)}{dt} &= 2\sigma \vec{e}_1(t) - 2\sigma \vec{e}_2(t) \\
    \frac{d\vec{i}_1(t)}{dt} &= 2\sigma \vec{e}_2(t) - 2\gamma \vec{i}_1(t) \\
    \frac{d\vec{i}_2(t)}{dt} &= 2\gamma \vec{i}_1(t) - 2\gamma \vec{i}_2(t) \\
    \frac{d\vec{r}(t)}{dt} &= 2\gamma \vec{i}_2(t) \\
    \frac{d\vec{p}_0(t)}{dt} &= \vec{\lambda} \circ \vec{s}(t) - \omega_0 \vec{p}_0(t) \\
    \frac{d\vec{p}_1(t)}{dt} &= \omega_0 \vec{p}_0(t) - \omega_1 \vec{p}_1(t) \\
    \frac{d\vec{p}_2(t)}{dt} &= \omega_1 \vec{p}_1(t) - \omega_{2} \vec{p}_2(t) \\
    \frac{d\vec{p}_3(t)}{dt} &= \chi \omega_{2} \vec{p}_2(t) - \omega_{3} \vec{p}_3(t)
\end{aligned}
\label{SEIR:eq:fullODEs}
\end{equation}
where $\circ$ indicates element-wise multiplication and $\matr{N}$ is a matrix with $(\matr{N})_{aa'} = N_{a'} / N_{a}$.
See \cref{SEIR:eq:fullODEs}

I solve the system of ODEs using Euler's method, discretised at timesteps of $\delta t = 1/2$ days.
That is, if $\vec{x}_t$ is the system state at time $t$ for $t \in \{ 0, 0.5, 1, \dots \}$ then $x_{t+\frac{1}{2}} = x_t + (d\vec{x}(t)/dt) \delta t$.

The mean prevalence on day $t$, used for the likelihood (see \cref{SEIR:eq:likelihood}), can be calculated as $\bar{p}_{at} = \frac{1}{2} \sum_{k=1}^l ( p_{kat} + p_{ka,t+1/2} )$.

\subsubsection{Inference} \label{SEIR:sec:inference}

Following \textcite{birrellBayesian}, I reparameterize the model to improve identifiability.
I parameterize $\beta_1$ in terms of the initial growth of the epidemic.
I also use a parsimonious parameterization of the initial conditions using the initial proportion of the population that are infectious and the initial proportion that are susceptible only.

Both reparameterizations rely on an assumption that the epidemic is in a steady-state exponential growth phase at time 0 (see \cref{E-SEIR:sec:SIR}). During such an exponential growth phase, all components of the $\vec{e}$, $\vec{i}$ and $\vec{p}$ compartments grow at the same growth rate, $\psi$.
That is:
\begin{equation}
\begin{aligned}
\frac{d\vec{e}_1(t)}{dt} = \psi \vec{e}_1(t) \\
\frac{d\vec{e}_2(t)}{dt} = \psi \vec{e}_2(t) \\
\frac{d\vec{i}_1(t)}{dt} = \psi \vec{i}_1(t) \\
\frac{d\vec{i}_2(t)}{dt} = \psi \vec{i}_2(t) \\
\frac{d\vec{p}_0(t)}{dt} = \psi \vec{p}_0(t) \\
\frac{d\vec{p}_1(t)}{dt} = \psi \vec{p}_1(t) \\
\frac{d\vec{p}_2(t)}{dt} = \psi \vec{p}_2(t) \\
\frac{d\vec{p}_3(t)}{dt} = \psi \vec{p}_3(t).
\end{aligned}
\label{SEIR:eq:exponential-growth}
\end{equation}
This follows directly from \cref{SEIR:eq:fullODEs} if $\vec{s}$ is constant. 

The growth rate $\psi$ is easily identifiable from the data, and therefore a parameterization in terms of $\psi$ should be more computationally efficient.
If this is the initial exponential phase, when $\vec{s} \approx 1$, then the basic reproduction number $\R$ for an SEIR model is linked to this growth rate by~\autocite{birrellBayesian,wearingAppropriate}:
\begin{align}
    R_{\psi} = \frac{\psi \left( \frac{\psi}{2\sigma} + 1 \right)^2}{\gamma \left(1 - \frac{1}{\left(\frac{\psi}{2 \gamma} + 1 \right)^2} \right)} \label{SEIR:eq:rtoR}.
\end{align}

Using the results of \cref{E-SEIR:sec:structured-populations}, $\R$ can also be written as $R_{\beta} = \beta_1 D(\matr{\kappa}(t) \circ \matr{q} \circ \matr{N})$ (recall that $D(\matr{\beta})$ denotes the dominant eigenvalue of $\matr{\beta}$).
Hence, I set $\beta_1 = R_\psi / D(\matr{\kappa}(t) \circ \matr{q} \circ \matr{N})$ to give $R_\psi = R_\beta$.

To form a parsimonious parameterization of the initial state, first focus on the initial proportion of the population that is in the infectious compartments.
Recall, from \cref{E-SEIR:sec:structured-populations}, that in the exponential growth phase, the relative proportion of each stratum that is infectious is proportional to the dominant eigenvector of $\matr{\beta}$, which I denote $\vec{d}$.
Introduce a model parameter $i^+$, the proportion of the population in either of the $i$ states at time 0 in the strata with the highest proportion infectious.
Let $\vec{i}_0 = \vec{i}_1(0) + \vec{i}_2(0)$ be the total number of infectious individuals at time zero.
Then:
\begin{align}
\vec{i}_0 = \frac{i^+}{\max \vec{d}} \vec{d}.
\label{SEIR:eq:i0}
\end{align}

Equating the expressions for $d\vec{i}_2/dt$ in \cref{SEIR:eq:fullODEs} and \cref{SEIR:eq:exponential-growth} and setting $t=0$ to allow substituting in \cref{SEIR:eq:i0} gives:
\begin{align}
    \vec{i}_1(0) &= \vec{i}_0 \left(1 + \frac{2\gamma}{\psi + 2\gamma} \right)^{-1}.
\intertext{By definition of $\vec{i}_0$:}
    \vec{i}_2(0) &= \vec{i}_0 - \vec{i}_1(0).
\end{align}

Equating the expressions in \cref{SEIR:eq:fullODEs} and \cref{SEIR:eq:exponential-growth}, gives the following solution for the initial state of several compartments.
\begin{align}
    \vec{e}_2(0) &= \vec{i}_1(0) \frac{\psi + 2\gamma}{2\sigma} \\
    \vec{e}_1(0) &= \vec{e}_2(0) \frac{\psi + 2\sigma}{2\sigma} \\
    \vec{p}_0(0) &= \vec{e}_1(0) \frac{\psi + 2\sigma}{\psi + \omega_0} \\
    \vec{p}_1(0) &= \vec{p}_0(0) \frac{\omega_0}{\psi + \omega_1} \\
    \vec{p}_2(0) &= \vec{p}_1(0) \frac{\omega_1}{\psi + \omega_{2}} \\
    \vec{p}_3(0) &= \vec{p}_2(0) \frac{\pi \omega_{2}}{\psi + \omega_{3}}.
\end{align}

Define a parameter $\vec{\pi}$ as the proportion of each stratum without initial immunity, then $\vec{r}(0) = 1 - \vec{\pi}$.
Finally, to ensure the population proportions sum to one, $\vec{s}(0) = \vec{\pi} - \vec{i}_0 - \vec{e}_1 - \vec{e}_2$.

I compute the posterior distribution using MCMC.
Specifically, I use an adaptive random-walk Metropolis--Hastings algorithm (discussed in \cref{E-MCMC}).
The proposal distribution is a multivariate normal that adapts to the covariance of the posterior.
The adaptive algorithm is based on \textcite[algorithm 4]{andrieuTutorial}, with the implementation based on \textcite{ghoshApproximate}'s work.
Proposals are accepted or rejected using the standard Metropolis--Hastings (MH) acceptance probability.
See \cref{SEIR:MCMC-algorithm} for full details.

\begin{algorithm}
 Set $\vec{X}_0$ to an initial value of the parameter vector \;
 $\vec{\mu_0} = \vec{X}_0$ \;
 $\matr{\Sigma_0} = \text{diag}(\vec{\mu_0})$ \;
 $\lambda_0 = 1$ \;
 \For{$i = 1, \dots, M$}{
  sample $\vec{Y_}i \sim \N(\vec{\mu_{i-1}}, \lambda_{i-1}\matr{\Sigma_{i-1}})$\;
  set $\vec{X_i}$ to $\vec{Y_i}$ or $\vec{X_{i-1}}$ using a MH acceptance step\;
  update the proportion of proposals accepted so far, $\alpha$ \;
  \eIf(\tcc{No adaptation for 200 iterations}){$i \leq 200$}{
    $\lambda_i = \lambda_{i-1}$ \;
    $\vec{\mu_i} = \vec{\mu_{i-1}}$ \;
    $\matr{\Sigma_i} = \matr{\Sigma_{i-1}}$ \;
   }{
    $\gamma_i = (i - 200)^{-0.6}$ \;
    $\log \lambda_i = \log \lambda_{i-1} + \gamma_i(\alpha - 0.234)$ \;
    $\vec{\mu_i} = (1 - \gamma_i) \vec{\mu_{i-1}} + \gamma_i \vec{X_i}$ \;
    $\matr{\Sigma_i} = (1 - \gamma_i) \matr{\Sigma_{i-1}} + \gamma_i (\vec{X_i} - \vec{\mu_i})(\vec{X_i} - \vec{\mu_i})^T$ \;
  }
 }
 \caption{Algorithm for adaptive random-walk Metropolis--Hastings. $\vec{\mu_i}$ and $\matr{\Sigma_i}$ are an estimate of the mean and covariance of the posterior distribution using information up to iteration $i$. $\text{diag}(\vec{\mu_0})$ is the diagonal matrix with diagonal entries equal to $\vec{\mu_0}$. $\lambda_i$ is the scale parameter of the proposal distribution at iteration $i$, tuned to try and ensure an optimal proportion of proposals are accepted (23.4\%). $\gamma_i$ is the learning rate, which determines how much adaptation occurs. $\gamma_i \to 0$ as $i \to \infty$ so the rate of adaptation is \emph{vanishing}. Vanishing adaptation guarantees the algorithm converges to the target distribution~\autocite[section 3]{andrieuTutorial}.}
 \label{SEIR:MCMC-algorithm}
\end{algorithm}

I transformed parameters with bounded supports to the real line to improve sampling efficiency.
Parameters which are constrained to the positive reals ($\sigma_\epsilon$, $\rho$, and $q_c$) were sampled on the log scale.
Parameters which are constrained to the unit interval ($i^+$ and $\vec{\pi}$) were sampled on the logit scale.

All results are based on four Markov chains each of length \numprint{1000000} iterations.
The first \numprint{500000} were discarded as burn-in, with the remaining chains thinned to every 100th iteration.
This gives \numprint{5000} posterior samples per region.
This was further thinned to \numprint{2000} samples when calculating posterior predictive quantities (incidence and prevalence) due to their computational cost.

\subsubsection{Priors} \label{SEIR:sec:priors}

Priors for the parameters are given in \cref{SEIR:table:priors}.
The parameters governing the duration of detectability were explained previously (see \cref{SEIR:table:ec-params}).

The initial conditions and transition rate parameters are unidentifiable in an SEIR model~\autocite{dankwaStructural}, therefore strongly informative priors or fixed values are used for them.

$\sigma_\epsilon$ and $\rho$ are given priors that favour parsimony.
The prior on $\sigma_\epsilon$ is a penalized complexity prior, favouring a null model of constant values for $\beta(t)$~\autocite{simpsonPenalising}.
The prior on $\rho$ favours a null model of a binomial likelihood.

To form the prior for $\vec{\pi}$, I used the analysis framework of \textcite{amirthalingamSeroprevalence}, applied to Aug 2020 seroprevalence data from blood donors~\autocite{amirthalingamSeroprevalence} and a representative sample of children~\autocite{ratcliffeCommunity}.
This analysis uses a Bayesian framework to correct the seroprevalence data for the sensitivity and specificity of the serological tests, using an informative prior.
\Textcite{amirthalingamSeroprevalence} formed the prior for the tests by evaluating the tests using samples which are known to be negative (taken before the pandemic) or positive (taken from an individual following a positive RT-PCR test).
A regression model predicts the proportion of the population with prior immunity in each age group and region.
I approximated the posterior of this proportion with a beta distribution to form the prior for the analysis in this chapter (see \cref{SEIR:table:immunity-prior}).

The remaining parameters have weakly informative priors.

\begin{landscape}
\begin{table}
\begin{tabular}{l c l l}
    Parameter description & Symbol & Distribution & Comment \\
    \hline \\
    Mean latent period & $1/\sigma$ & 3.5 days (fixed) & Based on \textcite{zhaoEstimating} \\
    Mean infectious period & $1/\gamma$ & 4 days (fixed) & Based on \textcite{zhaoEstimating} \\
    Initial proportion without immunity & $\vec\pi$ & See \cref{SEIR:table:immunity-prior} & \\
    Initial proportion infectious & $i^+$ & $\BetaDist(0.5, 1000)$ & Weakly informative \\
    Susceptibility of children & $q_c$ & $\LN(-0.4325, 0.1174)$ & Based on \textcite{vinerTransmission}  \\
    Initial growth rate & $\psi$ & $\N(0.06, 0.04)$ & Weakly informative \\
    Standard deviation of random walk steps on log scale & $\sigma_\epsilon$ & $\Exponential(80)$ & penalized complexity prior \\
    Overdispersion of observations & $\rho$ & $\Exponential(2 \times 10^5)$ & penalizing deviations from binomial
\end{tabular}
\caption[SEIR model priors]{Priors for each parameter. For details of the distributions and their parameterizations see \cref{E-distributions}.}
\label{SEIR:table:priors}
\end{table}
\begin{table}

\centering
\begin{tabular}{l|lllll}
         & 0--11 & 12--16 & 17--25 & 26--50 & 71+ \\
        \hline
        East Midlands & $\BetaDist(301, 8.5)$ & $\BetaDist(256, 18)$ & $\BetaDist(475, 19)$ & $\BetaDist(493, 20)$ & $\BetaDist(337, 5)$ \\
        East of England & $\BetaDist(236, 7.9)$ & $\BetaDist(248, 21)$ & $\BetaDist(677, 35)$ & $\BetaDist(613, 26)$ & $\BetaDist(332, 5.6)$ \\
        London & $\BetaDist(131, 9.7)$ & $\BetaDist(328, 50)$ & $\BetaDist(1331, 154)$ & $\BetaDist(990, 106)$ & $\BetaDist(204, 7.6)$ \\
        North East & $\BetaDist(180, 6.1)$ & $\BetaDist(145, 12)$ & $\BetaDist(274, 14)$ & $\BetaDist(295, 14)$ & $\BetaDist(240, 4.2)$ \\
        North West & $\BetaDist(252, 12)$ & $\BetaDist(353, 38)$ & $\BetaDist(885, 72)$ & $\BetaDist(695, 44)$ & $\BetaDist(264, 6.3)$ \\
        South East & $\BetaDist(404, 8.9)$ & $\BetaDist(319, 17)$ & $\BetaDist(577, 20)$ & $\BetaDist(586, 19)$ & $\BetaDist(356, 4.4)$ \\
        South West & $\BetaDist(415, 6.9)$ & $\BetaDist(277, 11)$ & $\BetaDist(556, 12)$ & $\BetaDist(559, 11)$ & $\BetaDist(492, 4)$ \\
        West Midlands & $\BetaDist(167, 8)$ & $\BetaDist(151, 19)$ & $\BetaDist(475, 32)$ & $\BetaDist(504, 33)$ & $\BetaDist(241, 5.8)$ \\
        Yorkshire and The Humber & $\BetaDist(314, 9.2)$ & $\BetaDist(242, 19)$ & $\BetaDist(589, 27)$ & $\BetaDist(474, 18)$ & $\BetaDist(316, 4.9)$ \\
    \end{tabular}
\caption{Priors for $\vec{\pi}$, the proportion of the population without prior immunity, in each age group and region combination.}
\label{SEIR:table:immunity-prior}
\end{table}
\end{landscape}

\subsubsection{Implementation}

I implemented the simulations study and inference code in Python~3.8.2~\autocite{python} using pandas~2.0.3~\autocite{pandas}, numpy~1.24.4~\autocite{numpy}, and scipy~1.10.1~\autocite{scipy}.
The strata are larger than those in \cref{backcalc}, allowing the data to be extracted from the SRS.
Therefore, inference was performed using the University of Cambridge high-performance cluster.

\subsection{Simulation study} \label{SEIR:sec:sim-study}

I ran a simulation study to study the inference procedure's performance and the model's practical identifiability.
I generated 100 simulated datasets representing the East of England using the following procedure.

First, I drew 100 parameter sets drawn from the priors (see \cref{SEIR:table:priors}), except that the prior on $\psi$ was $\LN(0.048, 0.0035)$.
I used this more informative prior for the simulation study to restrict the simulations to epidemics growing at $t=0$.
The weakly informative prior, used for the main analysis, allows a wide range of epidemics including some that are shrinking at the start of the time period.
The prior for $\vec{\pi}$ was the same as the East of England.

Next, I solved the ODE system in \cref{SEIR:eq:fullODEs} for each parameter set.
All other aspects of the ODEs, including the data-informed contact matrices, $\matr{\kappa}$, are as previously described in this section.
Population sizes were those of the East of England, and the time period the same as the main analysis.
This means that the simulations created realistic incidence and prevalence curves (see \cref{SEIR:fig:sim-data}).
In particular, the October half-term school holiday is visible, especially in the school-age groups and the November lockdown is visible to some extent in all age groups.
\begin{figure}
    \thisfloatpagestyle{empty}
    \makebox[\textwidth][c]{\includegraphics[width=.9\paperwidth]{SEIR/sim/data}}
    \caption[Simulated data]{%
        (A) Incidence and (B) prevalence for the 100 simulated data sets as a proportion of each stratum.
        Prevalence shown is $\bar{p}_{kat}$, the true population prevalence on each day, excluding sampling noise.
        Each line represents one simulation.
    }
    \label{SEIR:fig:sim-data}
\end{figure}

Finally, I generated one dataset of test results for each parameter set.
These were generated according to the likelihood, \cref{SEIR:eq:likelihood}.
The daily number of tests was set to the mean of what CIS performed in the region over this period (\numprint{1630}), distributed across the population in proportion to its size.

Having generated 100 datasets, I used the inference procedure in \cref{SEIR:sec:inference} to find the posterior distribution conditional on each dataset.
Hence, I ended up with 100 parameter sets distributed according to the priors (the ``generative values"), with an associated dataset and posterior distribution for it.

To check the performance of the inference procedure, I calculated the coverage of the 50\%, 75\%, and 95\% central credible intervals for each posterior distribution.
Define coverage for a parameter as the proportion of credible intervals for that parameter containing the generative value.
As the parameters are drawn from the prior, if the posterior distribution is inferred correctly then the coverage for each parameter should equal the nominal probability (\eg the coverage should be 50\% for the 50\% credible intervals)~\autocite{cookValidation}.
This check is related to, although simpler than, simulation-based calibration~\autocite{taltsValidating}.
I constructed a confidence interval for the coverage using the Pearson-Klopper method implemented in the binom R package~\autocite{binom1-1}.

The majority of the confidence intervals contain the nominal probability (see \cref{SEIR:fig:sim-coverage}).
There is a tendency for the coverage to be slightly lower than the nominal probability, suggesting that the credible intervals are slightly too narrow.
This is a common issue with random walk MCMC (see \cref{E-MCMC:sec:improving}).
The same issue arises with the coverage for derived quantities incidence and prevalence (see \cref{SEIR:fig:sim-inc-prev}).
\begin{figure}
    \includegraphics{SEIR/sim/coverage}
    \caption[Coverage of simulation study]{%
        Point estimate and 95\% confidence interval for coverage of the 50\%, 75\%, and 95\% central credible intervals for each parameter; see text for full definition.
        Vertical lines show the nominal coverage.
    }
    \label{SEIR:fig:sim-coverage}
\end{figure}
\begin{figure}
    \makebox[\textwidth][c]{\includegraphics[width=0.9\paperwidth]{SEIR/sim/predictive_coverage}}
    \caption[Coverage of simulation study (derived quantities)]{%
        Point estimate and 95\% confidence interval for coverage of the 50\%, 75\%, and 95\% central credible intervals for (A) incidence and (B) prevalence; see main text for full definition.
        Horizontal lines show the nominal coverage.
    }
    \label{SEIR:fig:sim-inc-prev}
\end{figure}


In this set-up, the parameters $i^+$, $q_c$, and $\psi$ are fully identified; $\pi_0$ is not identified at all; the other parameters are partially identified, their posterior has moved away from the prior but there is insufficient information in the data to fully overcome the prior (see \cref{SEIR:fig:true-vs-posterior}).
The ability to identify $q_c$ is an advantage of using data which includes measurements on children, unlike data on hospitalisations or deaths.

For $\vec{\pi}$ and $\rho$, the correlation between the posterior median and the generative value is weak (see \cref{SEIR:fig:true-vs-posterior}).
This suggests that these parameters are not practically identifiable in this setting.
The posteriors align very closely with the priors (not shown) which reinforces this finding.
The lack of identifiability for $\vec{\pi}$ aligns with previous work showing this parameter is structurally unidentifiable, \ie not identifiable even in the limit of infinite data~\autocite{dankwaStructural}.
\begin{figure}
    \makebox[\textwidth][c]{\includegraphics{SEIR/sim/true_vs_posterior}}
    \caption[True vs posterior parameter values]{%
        True vs posterior median parameter values for each parameter in each simulation.
        The black line shows the line $y = x$ which the dots would lie on if the parameter is fully recovered.
        The blue line is a linear regression line.
    }
    \label{SEIR:fig:true-vs-posterior}
\end{figure}

\subsection{Application to CIS data} \label{SEIR:sec:application}

The 12--16 age group (roughly corresponding to compulsory secondary school education) is estimated to have the highest incidence (see \cref{SEIR:fig:incidence}).
Incidence is smooth within each week but shows discontinuities between weeks.
This is because the parameters within $\matr{\beta}(t)$ are constant within each week.
A constant $\matr{\beta}(t)$ would lead to exponential growth following a period of transitory dynamics, although the weekly switching may mean that these transitory dynamics never end, especially when incidence is high~\autocite{rhodesConvergence}.
The estimated timing of the peak in incidence varies by region, with the most common time being late December 2020 (see \cref{SEIR:fig:peak-incidence}).
In Yorkshire and the Humber, the October peak is estimated to be greater than the winter peak; for North West there is a significant probability on either being larger.
\begin{figure}
    \includegraphics{SEIR/CIS/incidence}
    \caption[Posterior estimates of incidence]{%
        Posterior estimates (95\% CrI) of incidence proportion by region and age.
    }
    \label{SEIR:fig:incidence}
\end{figure}
\begin{figure}
    \includegraphics{SEIR/CIS/p_peak}
    \caption[Posterior estimates of peak incidence timing]{%
        Posterior probability of the peak incidence proportion for each region and age combination occurring on each day.
    }
    \label{SEIR:fig:peak-incidence}
\end{figure}

The $\beta_w$ parameters absorb all the time-varying changes in transmission not captured by Google mobility and school attendance data.
They trend downwards up until early December, then they start increasing (see \cref{SEIR:fig:beta-walk}).
Correlation is clear across the regions even though each region is estimated independently, suggesting underlying biological or epidemiological changes across regions.

\begin{figure}
    \includegraphics{SEIR/CIS/beta_walk}
    \caption[Posterior estimates of $\beta_w$]{%
        Posterior estimates $\beta_w$ for each week.
        Shown are ``violins'' where the width is proportional to the kernel-density estimated posterior density at that point.
        Values trend downwards until early December when an increase starts.
    }
    \label{SEIR:fig:beta-walk}
\end{figure}

The \emph{attack rate} is the proportion of the population that have been infected at least once.
In this model it is $1 - \vec{s}$, the proportion susceptible at the time the study ends.
The attack rate is highest in the 12--16 age group in the North West at 68\% (95\% CrI: 66--70\%) (see \cref{SEIR:fig:attack-rates}).
Some of these infections occurred prior to the period covered by this study; during the period of this study 23\% (95\% CrI: 21--24\%) of the population in the North West aged 11--16 was infected.
\begin{figure}
    \makebox[\textwidth][c]{\includegraphics{SEIR/CIS/attack_rates}}
    \caption{Posterior median and 95\% CrI of the attack rate in each region and age group.}
    \todo[inline]{Maybe compare this to blood donors, as used in the prior.}
    \label{SEIR:fig:attack-rates}
\end{figure}

Overall, the model fits the data well (see \cref{SEIR:fig:prev,SEIR:fig:prev-appendix}).
There are a few features of the data which are not captured by the model and the posterior estimate appears to be a compromise with the prevalence too low in one age group and too high in another.
For example, in the North East region, the 12--16 age group has its prevalence overestimated in October but the 17--25 age group has its prevalence underestimated.
\begin{figure}
    % \thisfloatpagestyle{empty}
    % \vspace{-3cm}
    \makebox[\textwidth][c]{\includegraphics[width=.9\paperwidth]{SEIR/CIS/prev_main}}
    \captionsetup{width=0.8\paperwidth}
    \captionof{figure}[SEIR prevalence goodness-of-fit]{%
        Goodness-of-fit to prevalence data by region and age for selected regions (see \cref{SEIR:fig:prev-appendix} for others).
        The central ribbon and line show the posterior median and 95\% CrI estimate for the proportion of the relevant stratum RT-PCR positive on each day.
        The lighter outer ribbon is the 95\% posterior predictive interval for the proportion of test results that are positive each week.
        The dots show the observed prevalence in CIS, aggregated by week.
        For a well-calibrated model, 95\% of the dots would be within the outer ribbon.
    }
    \label{SEIR:fig:prev}
\end{figure}

All regions' MCMC chains converged, assessed using the Rhat statistic and ESS (as defined in \cref{E-MCMC:sec:convergence}).
The highest Rhat was 1.01 for $\sigma_\epsilon$ in London.
All parameters had ESS > \numprint{1000}, except $\sigma_\epsilon$ and $\psi$ in London, which had ESS of 450 and 950 respectively.

\subsection{Discussion} \label{SEIR:sec:discussion}

The results in this section show the feasibility of fitting a mechanistic model using only data on prevalence.
If the distributions for length-of-stay in each compartment are known, then the transmission parameters can be estimated.

% Unlike most prior work (see\todo{ref section with lit review}), this modelling is based on a data source that is representative of the general population.
% % Therefore, the results have fewer biases and informed by data to a greater extent than this work.
% There is little-to-no ascertainment bias in the CIS, unlike sources relying on testing behaviour (in the UK, pillar 2 testing).
% The CIS allows estimates of transmission in children, unlike sources relying on severe events that occur almost entirely in adults~\autocite{bhopalChildren}.
Using a mechanistic model provides additional understanding, compared to the approach in \cref{E-backcalc}.
For example, decomposing the change in transmission into changing contacts, $\matr{\kappa}(t)$, changing susceptibility $\vec{s}(t)$, and other factors, $\beta(t)$.
By understanding which of these components is changing, policies can be better targeted.
If it is shown that $\beta(t)$ decreases, this implies that there is less transmission for the same number of contacts.
% Such decreases might be beneficial as it allows society to operate in a more normal manner without increasing transmission.
% Increasing activity without increasing transmission is desirable because it enables reductions the disease burden without burdesome restrictions on individuals day-to-day lives.


The parameters $\beta_w$ absorb all the time-varying factors not captured by Google mobility and school attendance data.
Many factors contribute to the pattern in the $\beta_w$ values, which have minima near the start of December.
While this analysis cannot establish causation, several hypotheses are worth considering.

The decreases in $\beta_w$ over the autumn could be due to increased caution (\eg masking or stricter adherence to social distancing) as cases and deaths increased~\autocite{jarvisEffect} and/or other effects not captured by $\matr{\kappa}(t)$, such as the impact of local measures in the highest transmission areas~\autocite{scottCovid19}.
Local measures are not well captured by $\matr{\kappa}(t)$ because the matrix averages behaviour across the whole of England.
Allowing each region to have its own $\matr{\kappa}(t)$ may capture the effect of local measures more accurately, although their targeted nature may require more granular methods.

The increases in $\beta_w$ in December coincide with the rise of the Alpha variant~\autocite{lythgoeLineage}, known to be more transmissible~\autocite[e.g.][]{daviesEstimated}.
A multiple variant model might be able to capture this rise more accurately.
Such a model would stratify positive tests in the CIS by pre-Alpha or Alpha variant.
Then, each variant would be explicitly included in the model to fit to this data.

These hypotheses for the mechanism causing changes in $\beta_w$ are correlated across the country.
This can explain the similar patterns that emerge for $\beta_w$ in different regions (see \cref{SEIR:fig:beta-walk}), despite these being estimated independently (\ie there is no model structure which favours correlated values).
These correlations could be exploited to improve the precision of estimates with a more complex model which borrows information across regions, \eg a hierarchical model.

The current model could be more flexible with regards to the age distribution of infections.
This would better exploit one of the strengths of the CIS: the availability of data  on all age groups.
In some regions, there appears to be a trade-off between under and overfitting in different age groups (see \cref{SEIR:fig:prev}).
For example, in the North East, most of the data points in the 12--16 age group are below the median posterior estimate, while most of the data points in the 17--25 age group are above the median posterior estimate.

Currently, the age distribution of infections is determined by $\matr{\beta}$, and to a lesser extent $\vec{\pi}$, the former of which only contains one fitted parameter that affects the age distribution, $q_c$.
Increasing the number of parameters in $\matr{\beta}$ would be the simplest approach to increase the model's flexibility.

The only component of $\matr{\beta}$ that allows age-dependent heterogeneity in the temporal changes is $\matr{\kappa}$.
It is plausible that this would not capture all behavioural changes.
For example, the introduction and relaxation of control measures within schools.
$\matr{\kappa}$ only captures changes in in-school transmission through changes in school attendance, not incorporating the probability of transmission, which could be modelled as changes in $q_{aa}$ for $a$ of school age.
Similarly, there is evidence that $q_c$ is different for Alpha and pre-Alpha variants~\autocite{zhuRole}, which could again motivate these changing over time.

The weekly patterns in incidence are a limitation of the current parameterization.
The random walk and contact matrices both vary only on the weekly timescale.
A daily random walk could provide more flexibility here.
However, this would greatly increase the number of parameters.
An alternative would be a smoothly varying function or an approximation of a random walk, such as that proposed by \textcite{ghoshApproximate}.

This discussion makes clear that the results are limited by the number of parameters, which is currently limited by the inference algorithm~\citePersonalComms{Sanmitra Ghosh}.
The MCMC algorithm used tends to struggle with a large number of parameters.
Alternative proposal schemes, such as updating the parameters in blocks could help, or a more advanced algorithm such as NUTS (used elsewhere in this thesis).

Extending the analyses carried out here to the rest of the pandemic would clearly be of interest.
I chose the end of the period to coincide with the vaccination rollout.
Vaccination dramatically affects the transmission dynamics by introducing some additional immunity into the population.
Mechanistic models require explicit representation of this immunity.

\section{Comparison of results} \label{transmission:sec:comparison}

\Cref{transmission:fig:compare-regions} compares the regional incidence estimates between the two models in this chapter.
Generally, the estimates agree with the SEIR model having much narrower credible intervals.
The largest discrepancy is that the phenomenological approach model estimates a second peak in mid-January in several regions (most clearly in London, North West, and the East of England).
However, except in London, the credible intervals still overlap.
This is a specific case of a larger pattern: the SEIR model produces smoother estimates of incidence than the phenomenological model.
\begin{figure}
    \centering \includegraphics{transmission/compare-regions}
    \caption[Comparing each models estimate of regional incidence.]{%
        Incidence proportion in England and each region estimated by the two models discussed in this chapter.
        Posterior median and 95\% CrI shown.
        For the SEIR model, newly RT-PCR positive (entrances into the P1 compartment), rather than incidence (entrances into the E compartment) are shown for comparability with the phenomenological results.
    }
    \label{transmission:fig:compare-regions}
\end{figure}

These patterns can be interpreted as an instance of the common bias---variance trade-off.
The SEIR model imposes stronger assumptions about the epidemic dynamics.
The assumptions reduce the variance of the estimates, including by making them smoother, but if the assumptions are incorrect, bias is introduced.
Furthermore, the SEIR model does not fully adjust for the non-response bias.
In particular, it makes no adjustment for the under-representation of ethnic minority groups in the CIS.
Adjusting for more covariates would require further stratifying the population and finding an appropriate parameterization of the contact matrix.
This trade-off based on the strength of assumptions has long been known to apply in the backcalculation context~\autocite[e.g.][section 8.3]{brookmeyerBackcalculation}.
The SEIR model can be viewed as a backcalculation method where the transmission part of the model is a smoothing method.

The phenomenological method benefits from being able to be implemented in a fast, approximate inference package (R-INLA) that easily scales to many parameters, including random effects.
Therefore, it is far more flexible and can easily include time-varying effects, ethnicity, etc.
The SEIR model is limited by the inference procedure and the requirement to solve an ODE to evaluate the likelihood.
To incorporate a more flexible model, and reduce its bias, a more efficient inference procedure would be required.

Many of these issues are clearly demonstrated by the estimates for the North East (see \cref{transmission:fig:compare-NE}).
I previously observed the quality of the SEIR model's fit is variable between different age groups (see \cref{SEIR:fig:prev,SEIR:fig:prev-appendix}).
This is because the model is not flexible enough to capture the true age distribution of infections.
The overestimation of prevalence in the 12--16 age group and underestimation in the 17--25 age group is reflected when comparing incidence estimates to those from the phenomenological model, which have reduced bias.
\begin{figure}
    \centering \includegraphics{transmission/compare-NE}
    \caption[Comparing each models estimate of North East incidence proportion by age.]{%
        Incidence proportion in each age group in the North East estimated by the two models discussed in this chapter.
        Posterior median and 95\% CrI shown.
        For the SEIR model, newly RT-PCR positive (entrances into the P1 compartment), rather than incidence (entrances into the E compartment) are shown for comparability with the phenomenological results.
    }
    \label{transmission:fig:compare-NE}
\end{figure}

A limitation of both sets of results is neglecting false negative and false positive test results.
Including these would increase the estimates of incidence because false negatives are more common than false positives (see \cref{biology-data:sec:measurement-error}).
The observation model could be adapted to include these errors, but the results of \cref{imperf-test} suggest they may change over the course of an infection.

An extension of either (or both) models into the rest of the epidemic would be useful.
The main obstacle is knowledge of how the duration of detectability distribution changes, and the need to incorporate the changes into the model.
The SEIR model would also need to include the effect of vaccination on transmission.


\section{Conclusion}

The major contribution in this chapter is showing that transmission and incidence can be estimated using only data from the CIS.
A similar survey could, therefore, be considered as the primary form of surveillance for a future pandemic.
The advantage of such a set-up is the unbiased information across the entire population, including children.

The transmission model presented in this chapter could be extended to fully extract the information in the CIS (see \cref{SEIR:sec:discussion}).
This would require a more flexible model, which would likely require a more efficient inference procedure, such as NUTS.

\ifSubfilesClassLoaded{
  \appendix
  \subfile{transmission-appendix-INLA}
  \subfile{transmission-appendix-phase-type}
  \subfile{appendix-plots}
  \listoftodos
}{}

\end{document}